Dockerfile

```diff
@@ -5,19 +5,25 @@ FROM unipartdigital/udes-tester:14.0
 ADD https://codeload.github.com/unipartdigital/odoo-package-hierarchy/zip/${ODOO_VERSION} \
     /opt/odoo-package-hierarchy.zip
 USER root
+RUN echo ${ODOO_VERSION}
```

Comment:
Do we need this? Or just makes debugging easier?

Reply:
No, it was for debugging.  I'll probably drop all these Dckerfile changes and just cherry-pick @adamrpatrick's commit.

Reply:
Cherry-picked.